### ISSUE TITLE
Change percent logic is a failure condition, which is reverse from the norm

### ISSUE THREAD

**Issue by guxie** [sodadata/soda-core#1547]
`- change percent for row_count > 50%`
The condition above is meant to fail anything that is NOT above the 50% threshold.
All other checks are designed to indicate the success condition, where this is a definition of a failure condition.

I'd recommend updating it such that the above `change percent` check is specifically as a success condition, which is the norm in other checks.

**Comment by janet-can** [comment]
I agree, [the thread](https://soda-community.slack.com/archives/C038FFU79J5/p1660649983086299) we were troubleshooting on offers a little more context, but both of us had to stop and think and test to validate the logic to really understand it. Would be good to have a completely consistent paradigm of setting pass state.
